service/controller/v5/arm_templates/ignition_container_setup.json

```diff
@@ -0,0 +1,69 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "accessTier": {
+            "defaultValue": "Cool",
+            "type": "string"
+        },
+        "accountType": {
+            "defaultValue": "Standard_RAGRS",
+            "type": "string"
+        },
+        "blobContainer": {
+            "defaultValue": "ignition",
+            "type": "string"
+        },
+        "kind": {
+            "defaultValue": "BlobStorage",
+            "type": "string"
+        },
+        "supportsHttpsTrafficOnly": {
+            "defaultValue": true,
+            "type": "bool"
+        },
```

Comment:
Can parameters above become variables? I find it easier to reason about as I know those aren't configurable outside the template.

Reply:
The can, but variables seems to be explicitely mutable for me. And parameters with default values no

Reply:
> variables seem to be explicitly mutable

I don't understand. They can't be mutated once initialized. And it's enforced they can't be set from outside the template.